src/eddymotion/model/base.py

```diff
@@ -287,12 +304,12 @@ def fit(self, data, **kwargs):
         # Calculate the average
         self._data = avg_func(shells, axis=-1)
 
-    def predict(self, gradient, **kwargs):
+    def predict(self, **kwargs):
         """Return the average map."""
         return self._data
 
 
-class PETModel:
+class PETModel:  ## Should this inherit from BaseModel ??
```

Comment:
Okay, you were already advancing the issue I mention above. We do want a `BaseModel` for all possible models, so maybe what's missing is a `BaseDWIModel`

Reply:
Done in commit df51602. Resolving.